server/src/main/scala/Server.scala

```diff
@@ -0,0 +1,17 @@
+package server
+
+import akka.stream.ActorMaterializer
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+
+object Server extends App {
+  println("Starting server...")
+
+  implicit val system = ActorSystem("system")
+  implicit val materializer = ActorMaterializer()
+  import system.dispatcher
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Imports should be grouped together](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2594816634/issues/source?bid=3301982&fileBranchId=3306761#l12)